packages/dx-react-scheduler/src/plugins/appointment-form.jsx

```diff
@@ -21,13 +33,19 @@ const defaultMessages = {
   cancelCommand: 'Cancel',
 };
 
+const pluginDependencies = [
+  { name: 'EditingState', optional: true },
+  { name: 'Appointments', optional: true },
+  { name: 'AppointmentTooltip', optional: true },
+];
+
 export class AppointmentForm extends React.PureComponent {
   constructor(props) {
     super(props);
 
     this.state = {
       visible: props.visible,
-      appointment: props.appointment,
+      appointment: {} || props.appointment,
```

Comment:
Hm, this line looks strange. `appointment` is always `{}`;

Reply:
A default value for appointment is `undefined` because this case correctly work with [this](https://github.com/DevExpress/devextreme-reactive/pull/1468/files#diff-eb36d6af4dfa29e1519a9a3731f05b67R77)